File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        int answer = 0;
        
        for(char ch:control.toCharArray()){
            if(ch=='w'){
                n+=1;
            }
            if(ch=='s'){
                n-=1;
            }
            if(ch=='d'){
                n+=10;
            }
            if(ch=='a'){
                n-=10;
            }
        }
        
        answer=n;
        
        return answer;
    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    var answer = 0;
    
    for (let ch of control) {
        if (ch == 'w') {
            n += 1;
        }
        if (ch == 's') {
            n -= 1;
        }
        if (ch == 'd') {
            n += 10;
        }
        if (ch == 'a') {
            n -= 10;
        }
    }
    
    answer = n;
    
    return answer;
}